lib/tasks/search.rake

```diff
@@ -71,7 +71,7 @@ namespace :search do
       # NOTE on the small chance that another process re-auto-creates the index
       # we just deleted before we have a chance to create the alias, this next
       # call will fail.
-      move_alias_to(Content::ES_INDEX_NAME, new_index_name)
+      move_alias_to(Content::ES_INDEX_NAME, new_index)
```

Comment:
I think it was just a typo. Various people complained about this in Slack and had fixed it on their own.  This code was not always run, so it was probably just missed during testing.

The PR that merged this into master is here: https://github.com/edx/cs_comments_service/pull/219